packages/core/src/Chip/index.tsx

```diff
@@ -152,6 +129,7 @@ const Chip: React.FC<ChipProps> = ({
       onClick={onClick}
       style={style}
       href={href}
+      to={to || href}
```

Comment:
Should this be `to || null`? If `href` is present wouldn't it be applied to `to` no matter what? So then even `<a` or `<button` would have the `to` prop?

Reply:
Fair point...was thinking flexibility, but that could cause issues

Reply:
Changed to just {to}